app/src/main/java/quotesredo/App.java

```diff
@@ -65,14 +34,26 @@ public static void main(String[] args) throws IOException {
                 //Step 5: use BuffedReader to read the input(can also use starReade.read()
                 try(BufferedReader starBufferedReader = new BufferedReader(starStreamReader)){
                     String line = starBufferedReader.readLine();
-                    System.out.println(line);
+                    //System.out.println(line);
 
 
                     //Step 6 part 1: create class(es) to convert formatted JSON Java Objects (come back here when you are done )
                     //Step 6 par 2: use GSON to create your Java objects
                     Gson gson2 = new Gson();
-                    StarwarsPerson starwarsPerson = gson.fromJson(line, StarwarsPerson.class);
-                    System.out.println(starwarsPerson.toString());
+                    Quote apiQuote = gson.fromJson(line, Quote.class);
+
+
+                    //Write JSON to file
+
+                    File quoteFileOut = new File("./app/src/main/resources/recentquotes.json");
+                    try (FileWriter quoteFileWriter = new FileWriter(quoteFileOut,true)){
+                        gson2.toJson(apiQuote, quoteFileWriter);
```

Comment:
In `recentquotes.json`, all of the quotes are in an array. When you append `apiQuote` to the end of the file, it's outside the array, which means it can't be read by your other code (and also causes a "malformed JSON" exception).

Instead, take a look at the array of `Quote` objects you create in `Quote.generateLocalQuote()`. Could you convert `apiQuote` to a `Quote` object, add it to that array, and then write the whole thing back to `recentquotes.json`? 

That way, the new quote will end up inside the array of quotes without having to mess around with the formatting of the file itself.